### DISCUSSION TITLE
List should also work without stars

### DISCUSSION THREAD

**Discussion by harishb00** [community/community#16989] [Other Feature Feedback, Questions, & Ideas]
Hi Team. It will be good if we have **List** option seperated from **Stars**. Stars kind of means that the repo I am going to star is something great that I need to see it again or the repo can be my favorite. Without creating stars, I may need to organize my repo into lists. With current feature, I need to first star a repo to add it to a list. Starred repo can go to list but non-star repo can also be added to list so that we can organize our repo into list of categories.

**Comment by Rudxain** [comment]
This post is a very similar to already-existing posts (quasi-duplicate), but I like your opinion
